enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2490,6 +2492,17 @@ func (c *FirecrackerContainer) stopMachine(ctx context.Context) error {
 	return nil
 }
 
+func (c *FirecrackerContainer) closeVMExecConn() {
+	if c.vmExec.conn == nil && c.vmExec.err == nil {
+		return
+	}
+	if err := c.vmExec.conn.Close(); err != nil {
+		log.Infof("Failed to close vm exec connection: %s", err)
```

Comment:
nit: use `Ctx` variant so it gets associated with the execution; also it can be warning or error-level

```suggestion
		log.CtxErrorf(ctx, "Failed to close vm exec connection: %s", err)
```

Reply:
done